src/react/components/_dashboard/UpNext/UpNextItem.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import VisuallyHidden from '../../VisuallyHidden';
+import format from 'date-fns/format';
+import { toggleMyUpNext } from 'Redux/actions/myUpNextActions';
+import { googleAnalytics } from 'functions/googleAnalytics'
+import './UpNextItem.scss';
+import '../../../stylesheets/useful.scss';
+import '../../../stylesheets/buttons.scss'
+
+const propTypes = {
+  dispatch: PropTypes.func,
+  item: PropTypes.object.isRequired,
+  index: PropTypes.number.isRequired,
+  applicationLayer: PropTypes.string
+};
+
+const UpNextItem = ({ dispatch, item, index, applicationLayer }) => {
+  const ga = new googleAnalytics(applicationLayer);
+
+  const expandItem = (index, dispatch) => {
+    dispatch(toggleMyUpNext(index));
```

Comment:
You've done the work to add this action and data into Redux, but that may not be necessary. You can use the `useState` hook and avoid that complexity because the for the item being expanded is all local to this component

Reply:
@developish - The other UpNextItems in the collection have their state changed to collapse them. That's why I'm using Redux to trigger their re-rendering after the state change.